### PR TITLE
winget: add Hindi translation

### PR DESCRIPTION
**Pull request by TheMimikyu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages.hi/windows/winget.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# winget
+
+> विंडोज़ पैकेज प्रबंधक।
+> अधिक जानकारी: <https://learn.microsoft.com/windows/package-manager/winget>.
+
+- एक पैकेज इनस्टॉल करें:
+
+`winget install {{package}}`
```
Comment (blueskyson) [inline pages.hi/windows/winget.md:8]: Please translate all the "package"s. ref: [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)
```suggestion
`winget install {{पैकेज}}`
```

File: pages.hi/windows/winget.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`winget upgrade --all`
+
+- `winget` के साथ प्रबंधित इन्सटाल्ड पैकेजों की सूची बनाएं:
+
+`winget list --source winget`
+
+- किसी फ़ाइल से पैकेज आयात करें, या स्थापित पैकेज़ को किसी फ़ाइल में निर्यात करें:
+
+`winget {{import|export}} {{--import-file|--output}} {{path/to/file}}`
```
Comment (blueskyson) [inline pages.hi/windows/winget.md:32]:
```suggestion
`winget {{import|export}} {{--import-file|--output}} {{फ़ाइल/का/पथ}}`
```

File: pages.hi/windows/winget.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- विंगेट-पीकेजीएस(winget-pkgs) रिपॉजिटरी में पीआर(PR) सबमिट करने से पहले मैनिफ़ेस्ट को सत्यापित करें:
+
+`winget validate {{path/to/manifest}}`
```
Comment (blueskyson) [inline pages.hi/windows/winget.md:36]: Please translate path/to/manifest as well.

File: pages.hi/windows/winget.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`winget uninstall {{package}}`
+
+- पैकेज के बारे में जानकारी प्रदर्शित करें:
+
+`winget show {{package}}`
+
+- पैकेज की खोज करें:
+
+`winget search {{package}}`
```
Comment (kbdharun) [inline pages.hi/windows/winget.md:20]:
```suggestion
`winget uninstall {{पैकेज}}`

- पैकेज के बारे में जानकारी प्रदर्शित करें:

`winget show {{पैकेज}}`

- पैकेज की खोज करें:

`winget search {{पैकेज}}`
```

File: pages.hi/windows/winget.md (hunk 3 above)
Comment (kbdharun) [reply pages.hi/windows/winget.md:36]:
```suggestion
`winget validate {{प्रकट/का/पथ}}`
```
Feel free to reword "manifest".

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

File: pages.hi/windows/winget.md (hunk 4 above)
Comment (TheMimikyu) [reply pages.hi/windows/winget.md:20]: Thank you for your suggestions.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**